pkgs/development/mobile/androidenv/README.md

```diff
@@ -10,6 +10,7 @@ You may need to make yourself familiar with [tests](https://nixos.org/manual/nix
 
 ```shell
 $ export NIXPKGS_ALLOW_UNFREE=1
+$ export NIXPKGS_ACCEPT_ANDROID_SDK_LICENSE=1
```

Comment:
Could you please move this to a separate PR?  It is not really related to the `libgcc_s` issue and I'm not the right reviewer for it.

Reply:
Those changes was in the spirit of Boy Scout Rule:
> The Boy Scouts have a rule: “Always leave the campground cleaner than you found it”
but anyway! I revert them back as you requested. check https://github.com/NixOS/nixpkgs/pull/226489/commits/398f142623dee16188587d66c83375c5fddb739c